aws/database/rds/database.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_db_instance" "main" {
+  db_name    = var.db_name
+  identifier = "${terraform.workspace}-${var.tags.Project}-${var.db_name}"
+
+  engine                     = var.engine
+  engine_version             = var.engine_version
+  auto_minor_version_upgrade = true
+
```

Comment:
oh yes that makes a lot of sense to me, it might be that in prod we actually would want that to happen during the upgrade window, but we can always make this configurable.